.github/actions/RunAutomation/SubmitStabilityJob/run.ps1

```diff
@@ -0,0 +1,35 @@
+param (
+    [Parameter(Mandatory=$true)]
+    $runParameters
+)
+
+$repository = $runParameters.Repository
+$targetBranch = $runParameters.TargetBranch
+$workflowName = " CI/CD"
+$workflowRunTime = Get-Date
```

Comment:
```suggestion
$workflowRunTime = Get-Date -AsUTC
```

I think the `$workflowRun.createdAt` down is in universal time. Please double-check.

Reply:
As far as I can see, that is the case yes.